.github/workflows/icu_ci.yml

```diff
@@ -0,0 +1,80 @@
+# Copyright (C) 2016 and later: Unicode, Inc. and others.
+# License & terms of use: http://www.unicode.org/copyright.html
+#
+# GitHub Action configuration script for ICU continuous integration tasks.
+
+name: ICU CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: '*'
+
+jobs:
+
+  # ICU4J build and unit test
+  icu4j-build-and-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout and setup
+        uses: actions/checkout@v2
+        with:
+          java-version: 1.11
+          lfs: true
+      - name: Checkout lfs objects
+        run: git lfs pull
+      - name: ICU4J
+        run: |
+          cd icu4j;
+          ant init;
+          ant check;
+          find out/junit-results -name "*.txt" -exec grep -l FAILED {} \;
```

Comment:
I see the successful negative testing to prove that the failure mode is handled as expected ([after introducing an error for negative testing](https://github.com/gnrunge/icu/runs/1539231211?check_suite_focus=true); [after reverting error for happy case testing](https://github.com/gnrunge/icu/runs/1539317756?check_suite_focus=true))

I'm still puzzled by impact of the `find ...\;` line. My interpretation is that this line should print out all files in the output dir that failed.  I see what you say that it [comes over from the Travis script](https://github.com/unicode-org/icu/blob/bd9c25ba1a6fe3169f9e2d67556419826250c231/.travis.yml#L39-L40), but from the Travis script's heading `after-failure` and examining the output log, I suspect that this line is not actually executing in GHA. I think GHA is exiting early on the failure from the `ant check` line.

Since that line is not doing anything, maybe we can remove that line and put a TODO?  And maybe later try out what would happen if you change
```
...
          ant check;
          find out/junit-results -name "*.txt" -exec grep -l FAILED {} \;
```
into
```
...
          ant check ||
            (find out/junit-results -name "*.txt" -exec grep -l FAILED {} \; 
              && exit 1)
```

Reply:
I removed the line, migrated over from Travis CI originally but confirmed now that 'ant check' fails (returns non-0) when a test fails. It is a no-op in GHA.